wg-serving/proposals/llm_instance_gateway/README.md

```diff
@@ -0,0 +1,217 @@
+
```

Comment:
I didn't realize this before, but I think this should be opened under github.com/kubernetes-sigs/wg-serving/proposals, not kubernetes/community.  Reasoning - this is a proposal for wg-serving sponsorship of the idea, which then translates into another sig owning a repo for the project (in this case, sig-network) while wg-serving drives that process.

@SergeyKanzhelev @terrytangyuan @ArangoGutierrez is that consistent with your perspective?

Reply:
Yes, I agree that this should be opened under wg-serving repo.

Reply:
++